app/utils/update-lms-form-values.js

```diff
@@ -9,8 +9,8 @@ const updateLMSFormValues = (
   const elToolname = document.getElementsByName(selectorToolname)[0];
   const key = document.getElementsByName('resourcekey')[0];
   const instancename = document.getElementById('id_name');
-
-  
+  const ltiversion = document.getElementsByName('ltiversion')[0];
```

Comment:
are you sure that it will not fail if in the DOM there will be NO elements with name `ltiversion`?

Reply:
this element exists, and it will be filled when the use clicks on the course to be selected. So no issue here